playwright.config.ts

```diff
@@ -61,36 +78,46 @@ const config: PlaywrightTestConfig<TestOptions> = {
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "5.21-edge",
+        storageState: "tests/.auth/user.json",
       },
+      dependencies: ["setup-chromium"],
     },
     {
       name: "firefox:lxd-5.21-edge",
       use: {
         ...devices["Desktop Firefox"],
         lxdVersion: "5.21-edge",
+        storageState: "tests/.auth/user.json",
       },
+      dependencies: ["setup-firefox"],
     },
     {
       name: "chromium:lxd-latest-edge",
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "latest-edge",
+        storageState: "tests/.auth/user.json",
       },
+      dependencies: ["setup-chromium"],
     },
     {
       name: "firefox:lxd-latest-edge",
       use: {
         ...devices["Desktop Firefox"],
         lxdVersion: "latest-edge",
+        storageState: "tests/.auth/user.json",
       },
+      dependencies: ["setup-firefox"],
     },
     {
       name: "coverage",
       use: {
         ...devices["Desktop Chrome"],
         lxdVersion: "latest-edge",
+        storageState: "tests/.auth/user.json",
```

Comment:
This should be a reference to the same `authFile` constant we use in `auth.setup.ts`.

Reply:
Yeah agreed, decided to export `authFile` from `lxd-test.ts` since it's related to global setup.